### PR TITLE
update PCA

### PR DESCRIPTION
**Pull request by haoyueshuai**
Update summary:

1. FID, IID settings for phenotype file
2. codes for MWE
3. workflow adjustments for outliers detection
4.  outliers plotting/highlighting

### CODE REVIEW

**Comment by gaow** [comment]
Thanks @haoyueshuai but why was the section of MWE commands deleted?

**Comment by haoyueshuai** [comment]
@gaow I moved those MWE commands to a new notebook with output showing. And point to it with a link.

File: GWAS/PCA.ipynb [hunk 1]
```diff
@@ -233,112 +233,11 @@
     "kernel": "SoS"
    },
    "source": [
-    "## How to run on a MWE?\n",
```
Comment (gaow) [inline GWAS/PCA.ipynb:236]: @haoyueshuai our other notebooks do have MWE commands and actually piopeline code in one place. That helps with portability. I suggest restore this section unless you have a compelling reason to do it in another separate notebook -- the link you provide points to a private repo that not everyone has access to.

**Review by gaow — COMMENTED** [review]
@haoyueshuai our other notebooks do have MWE commands and actually piopeline code in one place. That helps with portability. I suggest restore this section unless you have a compelling reason to do it in another separate notebook -- the link you provide points to a private repo that not everyone has access to. 

Otherwise it looks fine.

**Pull request merged**